### PR TITLE
[RIS-262] [refactor] button v2

### PR DESCRIPTION
**Pull request by asaadam**
refactor button v2

### CODE REVIEW

File: uikit/buttonV2/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { FunctionComponent, ButtonHTMLAttributes } from "react";
+import ButtonPrimary from "./ButtonPrimary";
+
+type ButtonPrimaryProps = {
+    size?: "md" | "lg" | "xl";
+    type?: "fab" | "default";
+} & Omit<ButtonHTMLAttributes<HTMLButtonElement>, "size" | "type">;
```
Comment (math-marcellino) [inline uikit/buttonV2/Button.tsx]: I think we are missing the `variant` props in here

File: uikit/buttonV2/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            default: "px-6 h-16 heading-h6",
+        },
+    },
+    type: {
+        fab: "rounded-lg",
+        default: "rounded-full",
+    },
+    variant: {
+        primary: "bg-dark-neutral-primary text-dark-background-default hover:bg-dark-neutral-soft",
+        secondary: "bg-light-neutral-subtle/10 py-3 text-dark-neutral-primary hover:bg-light-neutral-subtle/20",
```
Comment (math-marcellino) [inline uikit/buttonV2/Button.tsx]: I think the `py-3` here is not necessary, because we already use fixed height

**Review by math-marcellino — CHANGES_REQUESTED** [review]
Also don't forget to put this component on storybook

File: uikit/buttonV2/Button.tsx (hunk 1 above)
Comment (asaadam) [reply uikit/buttonV2/Button.tsx]: ups

File: uikit/buttonV2/Button.tsx (hunk 2 above)
Comment (asaadam) [reply uikit/buttonV2/Button.tsx]: I just copy-paste from your latest code haha
oke on it

**Comment by asaadam** [comment]
> Also don't forget to put this component on storybook

okayy

File: uikit/buttonV2/Button.tsx (hunk 2 above)
Comment (math-marcellino) [reply uikit/buttonV2/Button.tsx]: oops my bad, I think I missed that one haha

Comment (asaadam) [reply uikit/buttonV2/Button.tsx]: updated ser

File: uikit/buttonV2/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { FunctionComponent, ButtonHTMLAttributes } from "react";
+import { cls } from "../../utils/classTrim";
+import ButtonPrimary from "./ButtonPrimary";
+
+type ButtonPrimaryProps = {
+    size?: "md" | "lg" | "xl";
+    type?: "fab" | "default" | "square";
+    variant?: "primary" | "secondary" | "discord" | "twitter";
```
Comment (math-marcellino) [inline uikit/buttonV2/Button.tsx]: LGTM sir, but I think we are still missing a variant for `docs` --> the pink one
image

Comment (asaadam) [reply uikit/buttonV2/Button.tsx]: damn, forgot about this

**Review by math-marcellino — APPROVED** [review]
LGTM!

**Pull request merged**